File: Classical/src/main/java/prv/zielony/transaction/analyst/classical/dto/CashOperationEvent.java
package prv.zielony.transaction.analyst.classical.dto;

import prv.zielony.transaction.analyst.classical.model.CashOperationType;

/**
 * Created by zielony on 27.02.16.
 */
public class CashOperationEvent {

    private Long accountId;

    private CashOperationType type;

    private double amount;

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public CashOperationType getType() {
        return type;
    }

    public void setType(CashOperationType type) {
        this.type = type;
    }

    public Long getAccountId() {
        return accountId;
    }

    public void setAccountId(Long accountId) {
        this.accountId = accountId;
    }
}

File: Modern/src/main/scala/prv/zielony/transaction/analyst/modern/events/transactions/CashOperationEvent.scala
package prv.zielony.transaction.analyst.modern.events.transactions

import java.util.Date

import prv.zielony.transaction.analyst.modern.model.CashOperationType


/**
 * Created by zielony on 14.02.16.
 */
case class CashOperationEvent(override val timestamp:Date, val transactionId:Long,
                                override val status:TransactionStatus, val accountId:Long, val amount:Double,
                              val operation:CashOperationType) extends TransactionEvent(timestamp,
                                transactionId, status);
